Segv running custom command

**Running the following command**   
```
  minikube-start:
    section: "Runtime"
    description: "Start minikube cluster"
    command: |-
      minikube start --cpus max --memory max --driver="docker"
      devspace use context minikube
```
I got:  
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x60 pc=0x1021a0c28]

goroutine 1 [running]:
github.com/loft-sh/devspace/cmd.(*RunCmd).LoadCommandsConfig(0x1400048a640, {0x102c94000, 0x103fb8000}, {0x102c8dab8, 0x14000324290}, 0x140005b2c60?, {0x102c9d7d8, 0x1400021a300})
	github.com/loft-sh/devspace/cmd/run.go:253 +0x108
github.com/loft-sh/devspace/cmd.(*RunCmd).RunRun(0x1400048a640, {0x102c94000, 0x103fb8000}, {0x140001a61d0, 0x1, 0x1})
	github.com/loft-sh/devspace/cmd/run.go:124 +0x2ec
github.com/loft-sh/devspace/cmd.NewRunCmd.func1(0x13?, {0x140005b2a68?, 0x102c9d7d8?, 0x101abe46c?})
	github.com/loft-sh/devspace/cmd/run.go:79 +0xc0
github.com/loft-sh/devspace/cmd.NewRunCmd.NewSpecificRunCommand.func3(0x14000749800?, {0x103fb8000, 0x0, 0x0})
	github.com/loft-sh/devspace/cmd/run.go:362 +0x4c
github.com/spf13/cobra.(*Command).execute(0x14000749800, {0x103fb8000, 0x0, 0x0})
	github.com/spf13/cobra@v1.6.0/command.go:916 +0x658
github.com/spf13/cobra.(*Command).ExecuteC(0x1400047ec00)
	github.com/spf13/cobra@v1.6.0/command.go:1040 +0x318
github.com/spf13/cobra.(*Command).Execute(...)
	github.com/spf13/cobra@v1.6.0/command.go:968
github.com/loft-sh/devspace/cmd.Execute()
	github.com/loft-sh/devspace/cmd/root.go:151 +0x184
main.main()
	github.com/loft-sh/devspace/main.go:17 +0x30
```

**When I run the commands separately there is no problem. The minikube cluster start and devspace use the correct context**  


**How to reproduce: (?)** 

My devspace.yaml:
```
version: version: v2beta1
commands:
   minikube-start:
      section: "Runtime"
      description: "Start minikube cluster"
      command: |-
        minikube start --cpus max --memory max --driver="docker"
        devspace use context minikube
```

**Local Environment:**  
- DevSpace Version: 6.3.3
- Operating System: mac
- ARCH of the OS: ARM64
**Kubernetes Cluster:**  
- Cloud Provider: other 
- Kubernetes Version: 
```
Client Version: version.Info{Major:"1", Minor:"27", GitVersion:"v1.27.3", GitCommit:"25b4e43193bcda6c7328a6d147b1fb73a33f1598", GitTreeState:"clean", BuildDate:"2023-06-14T09:53:42Z", GoVersion:"go1.20.5", Compiler:"gc", Platform:"darwin/amd64"}
Kustomize Version: v5.0.1
Server Version: version.Info{Major:"1", Minor:"27", GitVersion:"v1.27.4", GitCommit:"fa3d7990104d7c1f16943a67f11b154b71f6a132", GitTreeState:"clean", BuildDate:"2023-07-19T12:14:49Z", GoVersion:"go1.20.6", Compiler:"gc", Platform:"linux/arm64"}
```

**Thanks for your time and nice project ! :)**

--- a/cmd/run.go
+++ b/cmd/run.go
@@ -250,10 +250,12 @@ func (cmd *RunCmd) LoadCommandsConfig(f factory.Factory, configLoader loader.Con
 	}
 
 	// verify client connectivity / authn / authz
-	_, err = client.KubeClient().Discovery().ServerVersion()
-	if err != nil {
-		log.Debugf("Unable to discover server version: %v", err)
-		client = nil
+	if client != nil {
+		_, err = client.KubeClient().Discovery().ServerVersion()
+		if err != nil {
+			log.Debugf("Unable to discover server version: %v", err)
+			client = nil
+		}
 	}
 
 	// Parse commands

